src/HoneybadgerSourceMapPlugin.js

```diff
@@ -64,7 +64,13 @@ class HoneybadgerSourceMapPlugin {
 
     return reduce(chunks, (result, chunk) => {
       const sourceFile = find(chunk.files, file => /\.js$/.test(file))
-      const sourceMap = find(chunk.files, file => /\.js\.map$/.test(file))
+
+      // webpack 5 stores source maps in `chunk.auxiliaryFiles` while webpack 4
+      // stores them in `chunk.files`. This allows both webpack versions to work
+      // with this plugin.
+      const sourceMap = (chunk.auxiliaryFiles || chunk.files).find(file =>
```

Comment:
Looking at it, it seems that using `auxiliaryFiles` does help, but I'm hitting an error further down the process. I'll dig into it tomorrow, but it seems that this fix might not be enough for fix the Webpack 5 compatibility.

Reply:
Are you using the `WriteFilePlugin` ?

Im coming across a lot of issues related to it with Webpack 5.

Reply:
Not since Webpack 5, no, or at least not explicitly. We were able to remove it for our development workflow by using `devServer.writeToDisk=true`.

Reply:
I think it may still present the same issue. Does it happen if `writeToDisk=false` ?

Reply:
It still does.